### ISSUE TITLE
[All][TextBlock] Add VerticalAlignment property to TextBlock

### ISSUE THREAD

**Issue by scottlovegrove** [microsoft/AdaptiveCards#4743]
**What platform is your feature request related to? (Delete other platforms).**

* .NET HTML
* .NET WPF
* Android
* iOS
* JavaScript
* UWP

**Is your feature request related to a problem? Please describe.**
I would like to be able to have text appear horizontally *and* vertically centered. Right now, the only option for TextBlock is for horizontal alignment.

**Describe the solution you'd like**
Add an additional field to TextBlock for VerticalAlignment

**Additional context**
Basically the "Agreed" and "Applause" text I would like to be in the middle of the square
![image](https://user-images.githubusercontent.com/25244878/92471614-21c0da80-f1d0-11ea-8dea-a7c9a8eed789.png)

**Comment by almedina-ms** [comment]
not for 20.09, moving to 20.10 @RebeccaAnne

**Comment by dclaux** [comment]
@scottlovegrove this can already be accomplished by setting the `verticalContentAlignment` of the Container (or Column) to "center".

**Comment by scottlovegrove** [comment]
@dclaux Yeah, I discovered this literally late afternoon yesterday. Closing this as it does what I need.
